### ISSUE TITLE
"SQL editor error markers refresh" shows up every few seconds while typing (DB2)

### ISSUE THREAD

**Issue by sbredenhof** [dbeaver/dbeaver#35095]
### Description

when im typing a DB2 select query, an error popup comes up every few seconds showing this: "SQL editor error markers refresh" 

it only appears to happen when there is a UNION, and the second statement uses an EXISTS

[log.txt](https://github.com/user-attachments/files/16513740/log.txt)

### DBeaver Version

Community 24.1.4.202408041450

### Operating System

Windows 11 Pro 23H2

### Database and driver

DB2
com.ibm.db2:jcc:RELEASE
11.5.9.0

### Steps to reproduce

1. connect to a db2 database
2. add this to the current script:
`SELECT 1 test FROM dual
UNION
SELECT 1 FROM dual WHERE EXISTS (SELECT 1 FROM dual )`
3. try type anything else

**Comment by E1izabeth** [comment]
Thank you for bug report.

**Comment by gisoto** [comment]
May be due to "Problem markers with semantic error descriptions" feature added to SQL editor in 24.1.4. Downgraded for now to 24.1.3, and I've not experienced this error since then

**Comment by iannacone** [comment]
another test case is when typing the query with a `UNION` and the `SELECT`s fields number don't match yet:
```SQL
(select id, name, 1
from table1)
union
(select id, name
from table2)
```

**Comment by jeferod83** [comment]
This problem also affects me too. I'll downgrade DBeaver to avoid this problem, because it was very boring... thanks!

**Comment by joshbrown-bnet** [comment]
I get this problem too. Every couple seconds or two. I'll probably downgrade as well.

**Comment by rajrao** [comment]
I get this problem with this sql (v.24.1.4)
```sql
select 'a', 'b'
union all
select 'a'
```

**Comment by Andrey-Svetlichny** [comment]
Same problem in v.24.1.4 on MacOS

**Comment by bzajacc** [comment]
same problem when using PSQL `EXCEPT` query

v.24.1.4 on MacOS

**Comment by jpdeutn** [comment]
same here! running version 24.1.4.202408041450 on Linux Mint 21.1.
will be waiting 4 fix before downgrading

**Comment by E1izabeth** [comment]
We fixed the issue. You can try Early Access version https://dbeaver.io/files/ea/

**Comment by ben-nour** [comment]
> same problem when using PSQL `EXCEPT` query
> 
> v.24.1.4 on MacOS

Same issue but on Snowflake

**Comment by AnabasisXu** [comment]
same issue when using UNION ALL
Version 24.1.4.202408041450
Window 10

**Comment by ben-nour** [comment]
> > same problem when using PSQL `EXCEPT` query
> > v.24.1.4 on MacOS
> 
> Same issue but on Snowflake

This issue is fixed for me with the latest update (24.1.5.202408181605)

**Comment by E1izabeth** [comment]
Please create new issue and describe your case in details
